src/pages/groupComparison/GroupComparisonMutationsTab.tsx

```diff
@@ -55,8 +56,8 @@ export default class GroupComparisonMutationsTab extends React.Component<
         let activeTabId;
         if (this.props.store.userSelectedMutationMapperGene) {
             activeTabId = this.props.store.userSelectedMutationMapperGene;
-        } else {
-            activeTabId = this.props.store.activeMutationMapperGene!
+        } else if (this.props.store.activeMutationMapperGene) {
```

Comment:
is it possible for actigeMutationMapperGene and userSelectedMutationMapperGene to differ?

Reply:
yes there is a scenario where the userSelectedMutationMapperGene is undefined. i just changed this though as they should really be the same